source/xml/reading.md

```diff
@@ -381,11 +381,11 @@ their respective XML elements:
             // Borrowing a parser from the KeyValue class.
```

Comment:
does `$reader->parseInnerTree();` also return namespaced attributes? if so, we also need to change line 373